src/types.rs

```diff
@@ -58,7 +59,7 @@ impl<'r> Responder<'r, 'static> for CreateResponse {
 
 #[derive(Serialize)]
 pub(crate) struct ExecuteResponse {
-	pub(crate) username: String,
+	pub(crate) username: Vec<u8>,
```

Comment:
Why would we want to reply with bytes instead of a human-readable string? I find it easier to manually interact with the API when the fields are human friendly (when possible)

Reply:
Several reasons:
- This username field doesn't really need to be human readable. When a user calls this endpoint they would already have to be authenticated so there's little utility in giving them a reminder as to who they are. If the intention is for this to be a special verification field for users to check if they are properly authenticated *after* executing a transaction, then I think we should rethink that.
- The username returned here is pretty much *only* consumed by the `Murmur > proxy` extrinsic when we specify the proxy `name` field. We technically could remove the field from the response, as it is somewhat a convenience feature. Instead we could ask the user to inject the username a second time on `execute` since they already know their username (and would have it readily available in the dapp, so it could easily be passed as an argument to the murmur.js execute function - i.e. no manual user intervention is needed). This would only be injected to murmur.js in order to construct the proxy call, **not** all the way to the API (so signature wouldn't change to accept another username). With this change we can omit the username field entirely from this struct, but we also get some redundancy as murmur.js requires the username to be submitted more than once, which to me implies that we should then remove the `username` cookie, as it is totally redundant. 
- Also, the API is unlikely to be manually invoked, so human readability isn't a big issue in my mind. We have the CLI and API for these purposes, I would generally discourage using CUrl or other tools except for testing purposes (maybe we include that as a note in the docs).
- it allows murmur.js to be simplified, instead of doing that conversion of string to bytes, we just need to set         `response.username` when constructing the call, which feels more efficient to me than doing the conversions especially when nobody is reading that string with their eyes anyway. Otherwise, we convert Vec -> String -> Vec without any real reason.
- also simplifies the API slightly, not needing to convert the vec to string when outputting from the execute function